src/mmw/js/src/core/models.js

```diff
@@ -136,7 +150,7 @@ var TaskModel = Backbone.Model.extend({
             }),
             pollingDefer = $.Deferred();
 
-            startDefer.done(function() {
+            startDefer.done(handleNullInCallback(function() {
```

Comment:
Replace `done` with `then`. If `done` fails I think nothing happens, but if `then` fails it gets caught and goes to `fail`